pandas/tests/series/test_timeseries.py

```diff
@@ -949,3 +949,8 @@ def test_get_level_values_box(self):
         index = MultiIndex(levels=levels, labels=labels)
 
         assert isinstance(index.get_level_values(0)[0], Timestamp)
+
+    def test_tz_naive_and_tz_aware_mix(self):
+        s = Series([Timestamp('20130101'),
```

Comment:
add the issue number as a comment

do a full comparision with assert_series_equal, IOW construct the expected

move this test to 
test_constructors.py